frappe/integrations/doctype/gsuite_settings/gsuite_settings.gs

```diff
@@ -0,0 +1,49 @@
+function doGet(e){
```

Comment:
this is what will be installed in google apps script (https://script.google.com/)